CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Added
 
+* New Menu Api - [#7545](https://github.com/decidim/decidim/pull/7545)
```

Comment:
Could you move this to before the `### Added` section and explain what happened? See the changelog for v0.24.0.rc1 for an example:

https://github.com/decidim/decidim/blob/release/0.24-stable/CHANGELOG.md#upgrade-notes